gossip/config.go

```diff
@@ -105,7 +105,7 @@ type (
 
 		ExtRPCEnabled bool
 
-		RPCLogsBloom bool
+		RPCBlockExt bool
```

Comment:
Suggestion for the future: move `RPCBlockExt` to the new `ethapi.Config` struct and configure API by constructor `NewPublicBlockChainAPI(Backend, Config)`